### PR TITLE
Minicap update to support more devices (included Android 12)

### PR DESCRIPTION
**Pull request by pcrepieux**
This commit introduce a new option to use a new minicap
implementation by specifying --screen-grabber or the env
variable SCREEN_GRABBER. The current native minicap is the default
and is refered as 'minicap-bin'. The new minicap which is packaged
in an apk is usable with 'minicap-apk' and will be used as a
fallback.
At the moment, this apk is expected to be included in
minicap-prebuilt node package at this location
minicap-prebuilt/prebuilt/noarch/minicap.apk.

STFService.apk has also been updated to take into account Android 12

### CODE REVIEW

**Comment by pcrepieux** [comment]
CI check failed because of minicap-prebuilt dependency. It requires this PR https://github.com/DeviceFarmer/minicap/pull/15

File: lib/units/device/resources/service.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = syrup.serial()
       pathutil.vendor('STFService/wire.proto'))
 
     var resource = {
-      requiredVersion: '2.4.6'
+      requiredVersion: '2.4.7'
```
Comment (koral--) [inline lib/units/device/resources/service.js:20]: There is no such version on https://github.com/DeviceFarmer/STFService.apk repo.
Is it intended to use it here now?

**Review by koral-- — APPROVED** [review]

**Comment by koral--** [comment]
> CI check failed because of minicap-prebuilt dependency. It requires this PR [DeviceFarmer/minicap#15](https://github.com/DeviceFarmer/minicap/pull/15)

Minicap is published, now CI fails due to issues detected by linter.

File: lib/units/device/resources/service.js (hunk 1 above)
Comment (pcrepieux) [reply lib/units/device/resources/service.js:20]: That's the latest addition from https://github.com/DeviceFarmer/STFService.apk/pull/42
https://github.com/DeviceFarmer/STFService.apk/blob/b75975682a5e4d32c37881515dfc07b3467221d6/app/build.gradle#L29
I think we can release it now.

**Pull request merged**